src/background/background-process.js

```diff
@@ -208,9 +214,9 @@ export default class BackgroundProcess {
    * @param {TabScript} tab - A TabScript object that represents a tab and it desired state
    */
   setContentState (tab) {
-    this.messagingService.sendRequestToTab(new StateRequest(tab), 10000, tab.tabID).then(
+    this.messagingService.sendRequestToTab(new StateRequest(tab), 10000, Tab.getTabIDfromUnique(tab.tabID)).then(
       message => {
-        let contentState = TabScript.readObject(message.body)
+        let contentState = TabScript.readObject(tab)
```

Comment:
I'm not sure about this change.  Previously we were setting the state from the contents of the message response and now it is ignoring the message response.  Is that intended? Why?